src/constants/permissions.js

```diff
@@ -34,8 +34,10 @@ export default {
     PERMANENT_DELETE_USER: 'permanent_delete_user',
     UPLOAD_FILE: 'upload_file',
     GET_PUBLIC_LINK: 'get_public_link',
-    MANAGE_WEBHOOKS: 'manage_webhooks',
-    MANAGE_OTHERS_WEBHOOKS: 'manage_others_webhooks',
+    MANAGE_INCOMING_WEBHOOKS: 'manage_incoming_webhooks',
+    MANAGE_INCOMING_OTHERS_WEBHOOKS: 'manage_incoming_others_webhooks',
```

Comment:
I think this should probably be called `MANAGE_OTHERS_INCOMING_WEBHOOKS`.

Reply:
Fixed